data/test_domains/test_domain_from_directory_tree/skill_2_domain/domain_pt3.yml

```diff
@@ -1,4 +1,5 @@
 version: "2.0"
 
 intents:
+  - utter_skill_1
```

Comment:
What was the purpose of changing these domain files? Is there already a test for this scenario? If not, should we add it?

Reply:
These files were added to my previous work updating the way the Domain is created when there are multiple domain files. Previously the test created the files in memory, so here I simply moved them into a directory of their own as most other domain test data is contained in files. I've moved the intent from the first domain file to a subsequent file because I wanted to test that a domain would be successfully merged when the first domain file contained no intents. Before, if this was the case, merging would fail with a `KeyError` (the code would look for an `'intents'` key, which didn't exist. My fix addresses that scenario, hence I wanted to have no intents in the first domain file, and moved them from there, to this file.

Reply:
Just for my erudition, since it's the KeyError happens at the 'OR' statement, wouldn't missing intents in either domain cause a KeyError when checked?